### ISSUE TITLE
let users know that a primary tab has some secondary tabs that contain additional related information

### ISSUE THREAD

**Issue by yfrimanm** [patternfly/patternfly-design#1224]
## Requesting new features, enhancements, or design changes to PatternFly

Currently Primary tabs which have nested secondary tabs are not marked as such.
I was thinking that it may be beneficial to users to know which primary tab contains several more secondary tabs, so they can look over there when searching for specific info.

This can be a simple solution if we can add a caret icon next to the tab name to indicate that there are more nested tabs below that primary tab. 

Please view the detailed explanation and mockups here:
[https://docs.google.com/document/d/1zokCPRft9Pt9QSZSCNNk9hgPp2coMgzTy9cfIoGYQTA/edit?usp=sharing]

Thank you.

**Comment by mcarrano** [comment]
@yfrimanm thanks for submitting this request. We probably would not use the caret as you suggest since we already use that for a optional tab overflow affordance. See below.



![Image](https://user-images.githubusercontent.com/6619860/212381899-6812c32a-6886-4b01-9c3c-b565c89a1547.png)

I do however, see the value of some visual indication that a tab has sub tabs. I invite you to put this on the agenda for a future PatternFly design share meeting as we can have more discussion.

**Comment by yfrimanm** [comment]
Thanks @mcarrano, will do.

**Comment by yfrimanm** [comment]
Alternative options to the caret icon can be: 

1. angle down icon 
![TABS angle down icon](https://user-images.githubusercontent.com/44577420/214070039-8214a371-5981-46e6-ac8c-1746898aece3.png)
2. horizontal ellipsis icon
![TABS horizontal ellipsis](https://user-images.githubusercontent.com/44577420/214070104-093bcdf2-9f26-4d96-b7a1-38440fd6518a.png)
3. The secondary tabs are shown on hover state (like a flyout)
![TABS secondary tabs show on hover state like a flyout](https://user-images.githubusercontent.com/44577420/214069901-5319856e-a100-4ffe-95a8-e4db4eacadc5.png)

**Comment by yfrimanm** [comment]
4. angle up+angle down icon
![TABS angle up+down](https://user-images.githubusercontent.com/44577420/217581260-1b4582a6-4a77-4be7-95d5-ae86c1c53933.png)

**Comment by JustinXHale** [comment]
> @yfrimanm thanks for submitting this request. We probably would not use the caret as you suggest since we already use that for a optional tab overflow affordance. See below.
> 
> ![Image](https://user-images.githubusercontent.com/6619860/212381899-6812c32a-6886-4b01-9c3c-b565c89a1547.png)
> 
> I do however, see the value of some visual indication that a tab has sub tabs. I invite you to put this on the agenda for a future PatternFly design share meeting as we can have more discussion.

I understand that we use the caret icon bc we use it for tab overflow, but why cant the same icon complete similar actions? The same icon is used an accordion menu, such as the one in the left nav.

**Comment by mcarrano** [comment]
Yes, that's true @JustinXHale . But I'd worry if we are using the same Icon for different behaviors within the tab bar.

**Comment by yfrimanm** [comment]
Hi, I've updated the [doc](https://docs.google.com/document/d/1zokCPRft9Pt9QSZSCNNk9hgPp2coMgzTy9cfIoGYQTA/edit#heading=h.ulnwetdvnep6) with a couple of more options.
I can share them in the next call to gather more feedback.
Thank you.

**Comment by yfrimanm** [comment]
@mcarrano we are making a progress with the double chevron down option (#4), because we would like to include it for the upcoming release. Do we know if this can get implemented as a PF component somewhere in the near future?
Thanks.

**Comment by mcarrano** [comment]
@yfrimanm Right now the team is heads down on the PatternFly 5 release through June. The earliest we would be able to get this in will be Q3. If you feel in a good place with the design proposal for this, I'd suggest getting it on the agenda for an upcoming PF design share meeting where it can be reviewed by the larger design community.

**Comment by mcarrano** [comment]
Any updates on this @yfrimanm ?

**Comment by yfrimanm** [comment]
@mcarrano unfortunately no. 
The "temporary" implementation that was created internally, wasn't satisfying enough. 
As for now, we are waiting the PF team to handle this component enhancement.
